contrib/perflog

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright (C) 2021  FreeIPA Contributors see COPYING for license
+#
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import division
+
+from datetime import datetime
+import logging
+import re
+from statistics import mean
+
+from ipapython import admintool
+from ipalib.facts import is_ipa_configured
+
+
+TIME_RE = re.compile(
+    r'\[(?P<date>.*)\] \[.*\].* \[pid \d+:tid \d+\] \[remote .*\] '
+    r'ipa: DEBUG: \[jsonserver_session\] (?P<principal>\S+): '
+    r'(?P<command>\S+)/1\(.*\): SUCCESS etime=(?P<etime>\d+)'
```

Comment:
Only the successful commands are considered but the failures may also be interesting. For instance the following line:
`ipa: DEBUG: [jsonserver_session] admin@DOMAIN.COM: user_del/1(['user1'], version='2.242'): NotFound etime=275287415` also shows an etime. It could be evaluated separately (a mean value for successes, a mean value for failures).

Reply:
Yeah, I suppose there could be a hickup in exception handling that could cause a slow-down, I'll see about adding this.